frontend/src/pages/components/notesMenu.js

```diff
@@ -47,6 +47,12 @@ export default function CustomizedMenus() {
     setAnchorEl(null);
   };
 
+  const gatheredInformation = ["Assignment", "Stakeholder 1", "Stakeholder 2"]
```

Comment:
What is this component used for? It doesn't have to do with the gatheredInfo component does it?

Reply:
I am not sure where we will exactly use it. I think Song and Tim made this file. However, I still made the change to resolve the comment.